app/google-services.json

```diff
@@ -0,0 +1,29 @@
+{
+  "project_info": {
+    "project_number": "450230911525",
+    "project_id": "madafaker-43c30",
+    "storage_bucket": "madafaker-43c30.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:450230911525:android:63c7b255a714539c6135cc",
+        "android_client_info": {
+          "package_name": "com.bbuddies.madafaker"
+        }
+      },
+      "oauth_client": [],
+      "api_key": [
+        {
+          "current_key": "AIzaSyB1iMEvRzv-NzrkOXMx6JsfugwtcCe4C_E"
```

Comment:
TODO: We'll need to extract API key in separate file and mark it in .gitignore, so random people on internet can't use our key. Create low priority task in "Infrastructure" epic, please.